clinical-domain-agent/application.yaml

```diff
@@ -79,6 +83,21 @@
 #       # Role assigned to the user
 #       role: client
 
+### Spring OAuth2 Client Security Configuration
+##! https://medizininformatik-initiative.github.io/fts-next/configuration/
+#   security:
```

Comment:
Can we use `spring.security.oauth2.client:` here? Similar to `spring.ssl.bundle` above.